saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/ConstrainedClassifier.scala

```diff
@@ -132,25 +132,28 @@ abstract class ConstrainedClassifier[T <: AnyRef, HEAD <: AnyRef](val onClassifi
       )(o.asInstanceOf[T])
   }
 
-  /** Test with given data, use internally
+  /** Derives Test Instances from the data model.
```

Comment:
Updated. We do have `.` at the end of some comments. No strict style guide though.